### ISSUE TITLE
Placement Indicator file is corrupted

### ISSUE THREAD

**Issue by kelsosharp** [TheUnityWorkbench/tuw-arfoundation-demo#6]
I have tried to use the png file on both windows and MacOS, and the png file will no longer open on either OS, and it can no longer be imported into Unity. This is also an issue with the plane as well.

**Comment by da5idmeier** [comment]
Yeah it happens when I "git clone" the repo, but if you just download the raw files from https://github.com/TheUnityWorkbench/tuw-arfoundation-demo/tree/master/Assets/Demo - and use them, they work fine.

**Comment by ellemcfarlane** [comment]
I'm unsure if it's corrupt exactly. My issues with it are that when I clone the whole repo I receive the error:

Downloading Assets/Demo/AR Placement Indicator.png (8.6 KB)
Error downloading object: Assets/Demo/AR Placement Indicator.png (22d023b): Smudge error: Error downloading Assets/Demo/AR Placement Indicator.png (22d023b48bf92c1255e93744ff185768107fbb6e4bef661f9079c780d7128330): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.
